edgekv/examples/mpulse-currency-converter-with-edgekv/main.js

```diff
@@ -0,0 +1,88 @@
+/*
+(c) Copyright 2020 Akamai Technologies, Inc. Licensed under Apache 2 license.
+Purpose:  Convert multiple currencies to one to support mPulse
+*/
+
+import { ReadableStream, WritableStream } from 'streams'
+import { httpRequest } from 'http-request'
+import { createResponse } from 'create-response'
+import { TextEncoderStream, TextDecoderStream } from 'text-encode-transform'
+import URLSearchParams from 'url-search-params';
+import { EdgeKV } from './edgekv.js';
+
+async function getCurrency(request){
+	//in this example the currency code from the purchase order is pulled from a query parameter
+    let params = new URLSearchParams(request.query);
+    let currencyCode   = params.get('currency'); 
+    let default_value  = 1;
+    
+    try {
+      const edgeKv = new EdgeKV({namespace: "namespace-name", group: "group-name"});

```

Comment:
for this to work with the included access token file, the namespace identifier passed to the EdgeKV constructor here should be "name" not "namespace-name". The EdgeKV JS lib logic would then look for a token associated with "namespace-name" key in edgekv_access_tokens imported from "edgekv_tokens.js".

Reply:
namespace-name and group-name would be replaced by actual values, as would name and name. I can change it but maybe add a comment to replace them with actual values.